module.rb

```diff
@@ -0,0 +1,23 @@
+module MyEnumerable
+  def all?
+    each do |e|
+      return false unless yield(e)
+    end
+    true
+  end
+
+  def any?
+    each do |e|
+      return false if yield(e)
+    end
+    true
+  end
```

Comment:
- [ ] This method fails for two test cases. I think you should invert the boolean return value and it would work ok. It should return true if any `yield(e)` is true, otherwise, it should return false if none of `yield(e)` is true. Kindly fix this issue. :pray: 

![Screenshot from 2021-12-10 18-41-18](https://user-images.githubusercontent.com/64894927/145601143-9f425d51-7683-4f10-81ad-e4e45a55dc8f.png)